### DISCUSSION TITLE
Good night. please help, how to solve this?

### DISCUSSION THREAD

**Discussion by ajiksekawan** [microweber/microweber#920] [General]
Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.2".

**Comment by engr-sakil** [comment]
You must be install PHP latest version.
New Microweber is required php version more then 8.1.2.

**Comment by peter-mw** [comment]
Hello, please install latest version of XAMPP or update your PHP version

**Comment by ajiksekawan** [comment]
Oke thanks.

Pada tanggal Sen, 8 Agt 2022 16:35, Peter Ivanov ***@***.***>
menulis:
